coral/functions/generate_ha_name_function.py

```diff
@@ -214,6 +214,9 @@ def post_save(self, tile, request, context):
             display_name.append(hb_number)
         if show_hpg_number:
             display_name.append(hpg_number)
+            self.update_display_name_tile(
```

Comment:
I have one question about this logic it's redundant by the next operations. For example the display name is a list of the order that the numbers will be applied. If we update the display name with the hpg number and ha name. The next code will add the ha name to the list of display name and then another update is ran to overwrite this update with the collection of reference numbers and the display name